init.lua

```diff
@@ -289,7 +306,81 @@ minetest.register_chatcommand("inspect", {
 							 minetest.formspec_escape(desc).."]"..
 							 "button_exit[2.5,7.5;3,1;close;Close]"
 
-		fsc.show(name, formspec, {}, function() end)
+		show_formspec(name, formspec)
+		return true
+	end,
+})
+
+local function inspect_item(itemstack)
+	local desc = "==== count ====\n"
+	desc = desc .. indent(1, "count = " .. itemstack:get_count()) .. "\n"
+
+	local meta = itemstack:get_meta()
+	local metatable = meta:to_table()
+	desc = desc .. "==== meta ====\n"
+	desc = desc .. indent(1, "meta.fields = " .. adjusted_dump(metatable.fields)) .. "\n"
+	
+	local itemdef = itemstack:get_definition()
+	-- combine itemdef table with its "superclass" table
+	local combined_fields = {}
+	local nodedef_fields = {}
+	for key, value in pairs(getmetatable(itemdef).__index) do combined_fields[key] = value end
```

Comment:
No benefit of squashing this loop. Please split this line into multiple lines to make it more readable.